packages/server/lib/automation/cookies.ts

```diff
@@ -142,13 +146,19 @@ export class Cookies {
       return cookie
     })
 
-    debug('set:cookies %o', cookies)
+    debug(`${eventName} %o`, cookies)
 
-    return automate('set:cookies', cookies)
-    // .tap(console.log)
+    return automate(eventName, cookies)
     .return(cookies)
   }
 
+  // set:cookies will clear cookies first in browsers that use CDP. this is the
+  // same as set:cookies in Firefox, but will only add cookies and not clear
+  // them in Chrome, etc.
+  addCookies (cookies, automate) {
```

Comment:
```suggestion
  addCookies (cookies: AutomationCookie[], automate: (eventName: string, cookies: AutomationCookie[]) => Bluebird.Promise<AutomationCookie[]>) {
```

Reply:
Addressed in [`db8684a` (#22320)](https://github.com/cypress-io/cypress/pull/22320/commits/db8684ab6c3f49fda2961344dcb23972701465b9)